docs/admin/users/idp-sync.md

```diff
@@ -420,36 +420,25 @@ settings, a user's memberships will update when they log out and log back in.
    Depending on the OIDC provider, this claim may be called something else.
    Common names include `groups`, `memberOf`, and `roles`.
 
-1. Configure the Coder server to read groups from the claim name with the OIDC
-   organization field server flag:
-
-   ```sh
-   # as an environment variable
-   CODER_OIDC_ORGANIZATION_FIELD=groups
-   ```
-
 1. Fetch the corresponding organization IDs using the following endpoint:
 
    ```text
    https://[coder.example.com]/api/v2/organizations
    ```
 
-1. Set the following in your Coder server [configuration](../setup/index.md).
+1. As a Coder admin, go to **Settings** > **IdP organization sync**.
```

Comment:
Note: we should probably clarify that the user must be an Organization Admin or a site-wide User Admin.